content/chat/rooms/messages.textile

```diff
@@ -434,3 +457,23 @@ The deleted message response is identical to the structure of a message, with th
 | version | Set to the serial of the deletion action. |
 | timestamp | Set to the time the message was deleted. |
 | operation | Set to the details the actioning client provided in the request. |
+
+h2(#global-ordering). Ordering Chat Message Events
+
+Chat messages and update events are delivered in realtime to clients connected to a particular region in the order in which that region receives them. The order in which a given region receives these events may be different from the "global" order of events, i.e. the true time-based order in which events happened.
+
+Chat messages are uniquely identified by their @serial@ and may have multiple @versions@ as a result of edit and delete operations. Both @serial@ and @version@ are lexicographically sortable strings. This means they can be used to enforce a deterministic global ordering based on string comparison.
+
+h3(#ordering-new). Ordering New Messages
+
+If the @serial@ of one message occurs before another when lexicographically sorted, the first message is considered to have occurred globally before the other. If the @serial@ values are identical, the two messages are, in fact, the same message.
+
+The @Message@ object also has convenience methods @before@, @after@ and @equal@ which provide the same comparison.
```

Comment:
Worth linking out to the API refs for these?

Reply:
Done!